### PR TITLE
installsh b62b

### PR DESCRIPTION
**Pull request by nhooyr**
- docs: Defend install.sh
- INSTALL.md: Demonstrate minimal script to install standalone release

### CODE REVIEW

File: docs/INSTALL.md [hunk 1]
```diff
@@ -31,6 +32,37 @@ methods:
...
+But this does not mean the install script is insecure. There is no major flaw that
+the install script is more vulnerable to than any other method of manual installation.
+The most secure installation method involves a second independent entity, i.e your OS
+package repos or Go's proxy server.
+
+We're careful shell programmers and are aware of the many footguns of the unix shell. Our
+script was written carefully and with detail. For example, it is not vulnerable to partial
+execution and the entire script runs with `set -eu` and very meticulous quoting.
+
+It follows the XDG standards, installs d2 properly into a unix hierarchy path (defaulting
```
Comment (alixander) [inline docs/INSTALL.md:50]: D2

File: docs/INSTALL.md [hunk 2]
```diff
@@ -31,6 +32,37 @@ methods:
...
+The install script is not the most secure way to install d2. We recommend that if
+possible, you use your OS's package manager directly or install from source with `go` as
+described below.
+
+But this does not mean the install script is insecure. There is no major flaw that
+the install script is more vulnerable to than any other method of manual installation.
+The most secure installation method involves a second independent entity, i.e your OS
+package repos or Go's proxy server.
+
+We're careful shell programmers and are aware of the many footguns of the unix shell. Our
```
Comment (alixander) [inline docs/INSTALL.md:46]: Unix

File: docs/INSTALL.md [hunk 3]
```diff
@@ -31,6 +32,37 @@ methods:
...
+uninstall.
+
+Some other niceties are that it'll tell you if you need to adjust `$PATH` or `$MANPATH` to
+access d2 and its manpages. It can also install
+[TALA](https://github.com/terrastruct/tala) for you with `--tala`. You can also use it to
+install a specific version of `d2` with `--version`. Run it with `--help` for more more
+detailed docs on its various options and features.
+
+If you're still concerned, remember you can run with `--dry-run` to avoid executing
+anything permanent.
```
Comment (alixander) [inline docs/INSTALL.md:61]: it's never permanent. "to avoid executing any writes"?

**Review by alixander — CHANGES_REQUESTED** [review]

File: docs/INSTALL.md (hunk 1 above)
Comment (nhooyr) [reply docs/INSTALL.md:50]: No here we should stick to lowercase as I'm referring to the `d2` command not D2. I'll add backquotes.

**Review by alixander — APPROVED** [review]

**Pull request merged**